aml-development/frontend
# feat(notification): implement mark read/unread, acknowledge, and delete functionality

**Description**
Allow users to interact with Notifications.

**Acceptance Criteria**
Users can mark notifications as read
Users can delete notifications

**How to Test**
Pull `notification_functionality` from `frontend`

- Click on the notification icon in the header.  Ensure that all notifications are acknowledged.  If a notification is sent and is unacknowledged, clicking the icon should acknowledge the notification.
- Click on the 'X' icon in a notification.  Ensure the notification is deleted.
- Click on the circle icon in a notification.  Ensure the read status is toggled and the UI updates accordingly (circle to half-filled circle or vice versa)
